core/src/main/resources/jenkins/slaves/systemInfo/EnvVarsSlaveInfo/systemInfo.jelly

```diff
@@ -29,5 +29,5 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <t:propertyTable items="${it.envVars}" />
+  <t:propertyTable items="${it.envVarsFull}" />
```

Comment:
FTR the `it` refers to the `SlaveComputer` in this case, it's the `instance` that refer to the `SlaveSystemInfo`.

Reply:
Done